README.adoc

```diff
@@ -212,34 +230,12 @@ The following sections below describe the feature set of the Cloud Spanner Hiber
 [options="header"]
 |===
 | Unsupported Feature | Description
-| Foreign Key Constraints  | Hibernate will respect database constraints but cannot enforce the foreign key constraint because this feature does not exist in Spanner.
 | Catalog and schema scoping for table names | Tables name references cannot contain periods or other punctuation.
 | Column default values | The dialect does not currently set default values based on the `@ColumnDefault` annotation, because Cloud Spanner does not support column defaults in the DDL.
 | Big-decimal or arbitrary-precision numbers | The dialect does not support `java.math.BigDecimal` and `java.math.BigInteger` because Cloud Spanner does not provide native support for https://cloud.google.com/spanner/docs/storing-numeric-data[arbitrary-precision decimal numbers], such as `NUMERIC` or `DECIMAL`.
-| Simba JDBC Driver | Hibernate ORM with Cloud Spanner is officially supported only with the open-source https://cloud.google.com/spanner/docs/use-oss-jdbc[Cloud Spanner JDBC Driver]. It does not support the https://cloud.google.com/spanner/docs/partners/drivers[Simba JDBC driver] at this time.
 |===
 
-=== Constraints and Entity Relationships
-
-The dialect supports all of the standard entity relationships:
-
-- `@OneToOne`
-- `@OneToMany`
-- `@ManyToOne`
-- `@ManyToMany`
-
-These can be used via `@JoinTable` or `@JoinColumn`.
-
-The dialect also supports unique column constraints applied through `@Column(unique = true)` or `@UniqueConstraint`.
-In these cases, the dialect will create a unique index to enforce uniqueness on the specified columns.
-
-However, Cloud Spanner does not support foreign key constraints, so the dialect will not generate foreign keys during schema generation.
-Note that this does not affect how Hibernate works because Hibernate does not rely on the existence of constraints to perform its operations.
-
-The lack of foreign key constraints also means that database-side cascading deletes are not supported via the `@OnDelete(action = OnDeleteAction.CASCADE)`.
```

Comment:
This is still the case, isn't it?

Reply:
Added clarification in the docs that FKs are supported, however database-side cascading deletes are not supported because spanner does not support cascading-delete settings.